test-tools/src/main/java/org/terracotta/org/junit/rules/ExternalResource.java

```diff
@@ -0,0 +1,86 @@
+package org.terracotta.org.junit.rules;
```

Comment:
The root content of this class was lifted from JUnit but lacks attribution in the README.md for this module.  The unit tests associated with `ExternalResource` should also have been picked up.

Reply:
Looking at this further, I suggest picking up the version from JUnit 4.13 -- it seems to have addressed the problem.